tests/always/test_project_structure.py

```diff
@@ -163,37 +163,24 @@ class TestGoogleProviderProjectStructure(unittest.TestCase):
     }
 
     MISSING_EXAMPLES_FOR_OPERATORS = {
-        # Deprecated operator. Ignore it.
+        # Deprecated operators
         'airflow.providers.google.cloud.operators.cloud_storage_transfer_service'
         '.CloudDataTransferServiceS3ToGCSOperator',
-        # Deprecated operator. Ignore it.
         'airflow.providers.google.cloud.operators.cloud_storage_transfer_service'
         '.CloudDataTransferServiceGCSToGCSOperator',
-        # Deprecated operator. Ignore it.
         'airflow.providers.google.cloud.sensors.gcs.GCSObjectsWtihPrefixExistenceSensor',
-        # Base operator. Ignore it.
         'airflow.providers.google.cloud.operators.cloud_sql.CloudSQLBaseOperator',
-        # Deprecated operator. Ignore it
         'airflow.providers.google.cloud.operators.dataproc.DataprocSubmitHadoopJobOperator',
         'airflow.providers.google.cloud.operators.dataproc.DataprocInstantiateInlineWorkflowTemplateOperator',
-        # Deprecated operator. Ignore it
         'airflow.providers.google.cloud.operators.dataproc.DataprocScaleClusterOperator',
-        # Base operator. Ignore it
         'airflow.providers.google.cloud.operators.dataproc.DataprocJobBaseOperator',
-        # Deprecated operator. Ignore it
         'airflow.providers.google.cloud.operators.dataproc.DataprocSubmitSparkJobOperator',
-        # Deprecated operator. Ignore it
         'airflow.providers.google.cloud.operators.dataproc.DataprocSubmitSparkSqlJobOperator',
-        # Deprecated operator. Ignore it
         'airflow.providers.google.cloud.operators.dataproc.DataprocSubmitHiveJobOperator',
-        # Deprecated operator. Ignore it
         'airflow.providers.google.cloud.operators.dataproc.DataprocSubmitPigJobOperator',
-        # Deprecated operator. Ignore it
         'airflow.providers.google.cloud.operators.dataproc.DataprocSubmitPySparkJobOperator',
         'airflow.providers.google.cloud.operators.mlengine.MLEngineTrainingCancelJobOperator',
-        # Deprecated operator. Ignore it
         'airflow.providers.google.cloud.operators.mlengine.MLEngineManageModelOperator',
-        # Deprecated operator. Ignore it
```

Comment:
Now it is not easy to tell for operators that there are no examples. What do you think to create some variables to break down these operators?
```
BASE_OPERATORS = ['AAA', 'BB']
DEPRECATED_OPERATORS = ['AAA', 'BB']
MISSING_EXAMPLES_FOR_OPERATORS = [
  *BASE_OPERATORS, *DEPRECATED_OPERATORS,
  'CC', 'DD',
]
```

Reply:
I thought they are all deprecated, to be honest, rather than missing. But if that's the case, then yeah - absolutely we should split them indeed. Give me a moment.

Reply:
Done :) . Good catch. I hope I classifted them correctly :D